cfg/mcuconf.h

```diff
@@ -50,7 +50,7 @@
  * Peripheral driver settings
  */
 
-#define HT32_GPT_USE_BFTM0                  TRUE
+#define HT32_GPT_USE_BFTM0                  FALSE
```

Comment:
I am using not using GPT. Switching to a single while loop instead. It makes it easier to implement thread suspend. Blocking on mutex/semaphores does not work with GPT intervals. I would need to stop and restart GPT intervals all the time. With a single while loop it is much simpler.

Reply:
The only thing I would have something to say about this is that I used interrupt to ensure the time between on and off periods. Since with even slightly longer delay between columns could cause the LED current to rise well above the rated current. (I think with this setup the UART interrupt could cause the temporary suspension of this thread, which could cause some problem with the LEDs I think.)

Reply:
Just from me looking at the current draw, this seems fine to me right now, just putting down a note